components/Form.jsx

```diff
@@ -52,16 +53,17 @@ const Form = ({formData, forNewMovie = true}) => {
           ]);
         }
       } else {
-        setMenssage([])
-        router.push("/");
+        setMenssage([]);
+        return data.data;
+        //   router.push("/");
       }
     } catch (error) {
-      console.log(error)
-  }
-  }
+      console.log(error);
+    }
+  };
```

Comment:
You can extract putData into a js module in "lib/movieApi.js" file and call it as updateMovie()

Reply:
done